### ISSUE TITLE
Why create-react-app instead of npm install react ?

### ISSUE THREAD

**Issue by xnoja** [forlooptanzania/ride-my-way#8]
Why are we using `create-react-app` instead of `npm install react --save`, if React itself is a node module?

**Comment by BenMaruchu** [comment]
@noelnoja Yes you can install just react and use it as it is but create-react-app have taken care of configurations that you will have to do manually to have a react development environment.

so to mention fews stuffs you will have to setup

1. If you want to use es6 features you will have to transpile your codes to es5 so that they can run in browser environment
2. You will have to figure out how to bundle your app which can be a daunting task especially configuring tools like webpack

can learn more [here](https://reactjs.org/docs/create-a-new-react-app.html)

Hopefully I have tried to answer your question
